setup.py

```diff
@@ -27,7 +27,7 @@
     install_requires=[
         "setuptools>=0.9",  # required for PEP 440 version parsing
         "packaging",
-        "futures",
+        "futures==3.1.1",
```

Comment:
Maybe `futures<=3.1.1` to decrease the possibility of conflicts?

Reply:
That's reasonable - done.